main.js

```diff
@@ -31,6 +32,8 @@ function clickHandlerOutfits() {
     selectButton(buttonBackgrounds, backgrounds);
   } else if (event.target.classList.contains('save-button-js')) {
     saveOutfit();
+  } else if (event.target.classList.contains('far-js')) {
```

Comment:
We plan on changing this class name in the next push so that it's more descriptive of what it is.

Reply:
Has been changed to close-button-js